spec/lib/notify/otp_message_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "notify/otp_message"
+
+RSpec.describe Notify::OTPMessage do
+  subject(:message) { Notify::OTPMessage.new("+4477881234567", "123456") }
```

Comment:
If you're worried about accidently spamming someone with text messages, 07700 900000 might be a plan.

https://www.numbersupermarket.co.uk/fake-number-ranges-here-are-the-numbers-reserved-exclusively-for-tv-radio-shows/

Reply:
Heh! Yes, I believe the GOV.UK docs mention this explicitly!

In this case though, everything is going to the [`fake_client`](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1589/commits/af6ca35e7c5adb30d85054367c75c296a6b8616b#diff-f39dd1bca5a438cf92e99327023c748d23a600c0545c96b26107e96f5e00a6dcR8)

Reply:
Yep, here they are in a page on [smoke testing](https://docs.notifications.service.gov.uk/ruby.html#smoke-testing). It's entertaining to note the provenance, mind 😆

Reply:
Oh, didn't realise they'd got their own hooks on some of those numbers! Nice.